index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet">
+    <title>first capstone project</title>
+</head>
+<body>
+    <header> 
+        <nav>
+            <a href="http://127.0.0.1:5500/index.html" class="music_logo">
+                <img src="images/label.png" alt="">
+             </a>
+            <div class="hamburger_menu">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <ul class="mobile_menu">
+                <li><a href="http://127.0.0.1:5500/about.html">About</a></li>
+                <li><a href="#">concert</a></li>
+                <li><a href="#">ticket</a></li>
+                <li><a href="#">shows</a></li>
+                <li><a href="#">News</a></li>
+                <li><a class="nav_link_ticket" href="#"> Get ticket</a></li>
+            </ul>
+            <ul class="nav_link">
+                <li><a class="nav_link_line" href="#">About</a></li>
+                <li><a class="nav_link_line" href="#">concert</a></li>
+                <li><a class="nav_link_line" href="#">ticket</a></li>
+            </ul>
+        </nav>
+
+    </header>
+    <main>
+        <section class="homepage">
+            <div class="homepage_content">
+                <div class="header_content">
+                    <h3>"welcome  y'all"</h3>
+                    <h1 class="h1_image">Monsters of rock world concert 2023</h1>
+                </div>
+                <div class="paragraph">
+                    <p>A great concert celebrating multiple talents from various counties <br>  
+                        which is taking place in gotham city on 15th october</p>
+                </div>
+                <div class="paragraph_two">
+                    <h2>2023.10.15(THU) ~ 16(FRI)</h2>
+                    <p>@lugogo cricket oval </p>
+                </div>
+            </div>
+        </section>
+        <section class="work_section">
+            <div class="work_section_content">
+                <div class="main_program">
+                    <h2>Main program</h2>
+                    <div class="line_program"></div>
+                </div>
+                <div class="program_details_wrapper">
```

Comment:
- [ ] Kindly add more spacing to the sides of this wrapper and decrease the spacing on the top and bottom to match the design. 🥸️

**Your design**
![Screenshot from 2023-02-09 16-26-39](https://user-images.githubusercontent.com/17243325/217840030-b74bb595-6519-4077-a6d7-3f436ef5a9c3.png)

**Template**
![Screenshot from 2023-02-09 16-27-58](https://user-images.githubusercontent.com/17243325/217840135-bb49b46e-2fe9-4fd8-99ee-81904ec15a83.png)